modules/apps/29-fee/ibc_middleware_test.go

```diff
@@ -607,166 +607,220 @@ func (suite *FeeTestSuite) TestOnRecvPacket() {
 	}
 }
 
-// different channel than sending chain
 func (suite *FeeTestSuite) TestOnAcknowledgementPacket() {
 	var (
-		ack                    []byte
-		packetFee              types.PacketFee
-		originalBalance        sdk.Coins
-		expectedBalance        sdk.Coins
-		expectedRelayerBalance sdk.Coins
-		expLocked              bool
+		ack                 []byte
+		packetID            channeltypes.PacketId
+		packetFee           types.PacketFee
+		refundAddr          sdk.AccAddress
+		relayerAddr         sdk.AccAddress
+		expRefundAccBalance sdk.Coins
+		expPayeeAccBalance  sdk.Coins
 	)
 
 	testCases := []struct {
-		name               string
-		malleate           func()
-		expFeesDistributed bool
-		expPass            bool
+		name      string
+		malleate  func()
+		expPass   bool
```

Comment:
The `expPass` is not checked when the suite runs down below. Is that on purpose?

Reply:
It is. [See here](https://github.com/cosmos/ibc-go/pull/1526/files#diff-9da095e29ee5f2cd43cc0268993bd6b6aaa612509b97b6b559391b43b6dc6cbbL773-R831)

Reply:
Oh, sorry, I missed it. :)